aea/configurations/loader.py

```diff
@@ -181,6 +181,27 @@ def from_configuration_type(
         configuration_type = PackageType(configuration_type)
         return ConfigLoaders.from_package_type(configuration_type)
 
+    def _validate(self, json_data: Dict) -> None:
+        """
+        Validate a configuration file.
+
+        :param json_data:
+        :return: None
+        :raises ValidationError: if the file doesn't comply with the JSON schema.
+              | ValueError: if other consistency checks fail.
+        """
+        # this might raise ValidationError.
+        self.validator.validate(instance=json_data)
+
+        expected_type = self.configuration_class.package_type
+        # TODO 'type' is optional for backward compatibility
```

Comment:
keep in mind this before the release

Reply:
after the latest versions of all the packages will be updated, the `"type" in json_data` should be removed.